drkafka/src/main/java/com/pinterest/doctorkafka/servlet/DoctorKafkaInfoServlet.java

```diff
@@ -38,7 +38,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
       writer.print("<p> Version: " + version + ", Uptime: " + jvmUpTimeInSeconds + " seconds </p>");
       writer.print("</div>");
 
-      List<KafkaClusterManager> clusterManagers = DoctorKafkaMain.doctorKafka.getClusterManagers();
+      Collection<KafkaClusterManager> clusterManagers = DoctorKafkaMain.doctorKafka.getClusterManagers();
```

Comment:
If a cluster is in maintenance mode, shall we also make that info show up in DoctorKafka web UI?

Reply:
Sure, adding a new column to show maintenance mode status.
![image](https://user-images.githubusercontent.com/9707358/50244706-7598e980-0385-11e9-8104-b92fdbfb8b00.png)